macros/src/lib.rs

```diff
@@ -7,7 +7,10 @@ mod command;
 mod modal;
 mod util;
 
+use std::vec;
```

Comment:
Isn't this in the prelude?

Reply:
Pretty sure, yeah. Editor added that for some reason.